api/deployment.go

```diff
@@ -0,0 +1,35 @@
+// mystack-controller api
```

Comment:
do you need a deployment handler? it does not seems that we will expose it in the api